result.js

```diff
@@ -205,6 +205,11 @@ const methodToFunction = function (method) {
 };
 
 Object.keys(methods).forEach(function (key) {
+    // Breaks require in repl
```

Comment:
Please add a `NOTE(fmtq):` prefix to the comment. (Or whichever name you have there)